legend-engine-xt-sql-pure/src/main/resources/core_external_query_sql/binding/fromPure/fromPure.pure

```diff
@@ -143,13 +145,28 @@ function <<access.private>> meta::external::query::sql::transformation::queryToP
 {
   debug('processProjection');
 
-  let exp = if ($groupBy->isEmpty() && !anyColumnAggregate($select),
-    | processSelect($select, $context),
-    | processGroupBy($select, $groupBy, $context));
+  let aggregates = $select.selectItems->filter(si | $si->isSelectItemAggregate());
+  let windows = $select.selectItems->filter(si | $si->isSelectItemWindow());
+  let standard = $select.selectItems->filter(si | !($aggregates->contains($si) || $windows->contains($si)));
```

Comment:
Wont be simpler and mor efficient to just do a removeAll?  

`$select.selectItems->removeAll($aggregates)->removeAll($windows);`